pkgs/development/python-modules/pysaml2/default.nix

```diff
@@ -97,5 +97,6 @@ buildPythonPackage rec {
     changelog = "https://github.com/IdentityPython/pysaml2/blob/v${version}/CHANGELOG.md";
     license = licenses.asl20;
     maintainers = [ ];
+    broken = true;
```

Comment:
This should receive a comment with a reason

Reply:
`# See https://github.com/NixOS/nixpkgs/issues/367976 https://github.com/IdentityPython/pysaml2/issues/975` ?

Reply:
applied